components/Trade/Advanced/RightPanel/AccountDetails/index.tsx

```diff
@@ -221,7 +221,7 @@ const STable = styled(Table)`
     > thead {
         display: table;
         table-layout: fixed; /* even columns width , fix width of table too*/
-        width: calc(100% - 5px) !important; /* scrollbar is 5px */
+        width: 100%; /* scrollbar is 5px */
```

Comment:
```suggestion
        width: calc(100% - 5px) !important; /* scrollbar is 5px */
```
When the table has entries that scroll, without this then the head columns dont align with the table columns